apps/web/playwright/webhook.e2e.ts

```diff
@@ -67,12 +67,13 @@ test("add webhook & test that creating an event triggers a webhook call", async
   body.payload.eventTypeId = dynamic;
   body.payload.videoCallData = dynamic;
   body.payload.appsStatus = dynamic;
+  body.payload.metadata = dynamic;
```

Comment:
Do we want to redact metadata?

Reply:
I am not exactly sure what you mean with that?

Reply:
Redacted data applies for things we don't care what value have (like meeting urls) If we expected certain metadata to be present we may want to know that in the test.

Reply:
Updated snapshot to: "metadata":{"videoCallUrl":"[redacted/dynamic]"}